includes/flattened_custom_event.js

```diff
@@ -0,0 +1,152 @@
+const getKeys = (keys) => {
+    return keys.map(key => ({
+        [key.alias || key.keyName]: {
+          description: key.description,
+          bigqueryPolicyTags: key.hidden && key.hiddenPolicyTagLocation ? [key.hiddenPolicyTagLocation] : []
+        }
+    }))
+};
+module.exports = (params) => {
+    return params.customEventSchema.forEach(customEvent => {
+        publish(customEvent.eventType + "_" + params.eventSourceName, {

```

Comment:
AK to check: What does the 'publish' function do?

Reply:
@Arjun-Kharbanda it creates a table definition object. https://cloud.google.com/dataform/docs/develop-workflows-js